PassageFlex.podspec

```
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'PassageFlex'
-  s.version          = '0.3.0'
+  s.version          = '0.4.0'
```

Since its a minor bug fix, I recommend a patch version change :) 
```suggestion
  s.version          = '0.3.1'
```